app/frontend/wallet/shelley/build-transaction.ts

```diff
@@ -1,10 +1,7 @@
 import {computeRequiredTxFee} from './helpers/chainlib-wrapper'
-import _ from 'lodash'
-// import {Input, Output, TxPlan} from '../cardano-wallet'
+// import _ from 'lodash'
```

Comment:
I would remove this if it isn't used, no need to keep it. Btw, we don't have lodash in package.json, so we were relying on some dependency providing it